app/models/host.rb

```diff
@@ -22,7 +22,43 @@ class Host < ApplicationRecord
   validates :name, presence: true
   validates :account, presence: true
 
+  def in_inventory
+    host_in_inventory(id)
+  end
+
   class << self
+    def host_in_inventory(host_id)
+      ::HostInventoryAPI.new(
+        User.current.account,
```

Comment:
Even if we don't use this PR, I'd like to point out a couple things that we would have needed to fix.

Here's one - getting the current user from thread local memory like this should be done at the controller level and never at the model level. I know, User.current is used to store it, and User is a model, but the controllers (and graphql) are responsible for managing the state of who the current user is. graphql does not set `User.current` like REST does - it uses `context[:current_user]`, some kind of graphql context. Anyway, I don't know for sure if this would have caused issues, but it's something that worries me about this implementation.